cmd/rpcdaemon/commands/engine_api.go

```diff
@@ -78,6 +88,8 @@ type EngineAPI interface {
 	GetPayloadV1(ctx context.Context, payloadID hexutil.Bytes) (*ExecutionPayload, error)
 	GetPayloadV2(ctx context.Context, payloadID hexutil.Bytes) (*GetPayloadV2Response, error)
 	ExchangeTransitionConfigurationV1(ctx context.Context, transitionConfiguration *TransitionConfiguration) (*TransitionConfiguration, error)
+	GetPayloadBodiesByHashV1(ctx context.Context, hashes []libcommon.Hash) ([]*ExecutionPayloadBodyV1, error)
+	GetPayloadBodiesByRangeV1(ctx context.Context, payload BeaconBlocksByRangeRequest) ([]*ExecutionPayloadBodyV1, error)
```

Comment:
This method should have 2 parameters (`start`, `count`) per [the spec](https://github.com/ethereum/execution-apis/blob/main/src/engine/shanghai.md#engine_getpayloadbodiesbyrangev1), not one, similar to how `ForkchoiceUpdatedV1` has 2 parameters.